### PR TITLE
Feature/pupil spa lint fixes

### PR DESCRIPTION
**Pull request by jon-shipley**
Update readme
Lint fixes for pupil-spa and electron

### CODE REVIEW

**Comment by jon-shipley** [comment]
Looking for 1 approval

File: electron/lib/application-menu.js [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
 // empty menu
-const template = [];
+const template = []
 
-module.exports = function(app) {
-  if (process.platform == 'darwin') {
-    let name = app.getName();
+module.exports = function (app) {
+  if (process.platform === 'darwin') {
+    let name = app.getName()
```
Comment (GeorgeChatzigiannis) [inline electron/lib/application-menu.js]: guess its a const

File: electron/test/search.js [hunk 2]
```diff
@@ -1,74 +1,76 @@
-require('./hooks');
-var assert = require('assert');
+/* global describe it */
+
+require('./hooks')
+var assert = require('assert')
```
Comment (GeorgeChatzigiannis) [inline electron/test/search.js]: shouldn't be const?

Comment (jon-shipley) [inline electron/test/search.js]: Yes - that's tester code.  I'll update it.

**Pull request merged**